test/common/runtime/runtime_impl_test.cc

```diff
@@ -41,6 +41,20 @@ TEST(UUID, sanityCheckOfUniqueness) {
   EXPECT_EQ(num_of_uuids, uuids.size());
 }
 
+TEST(RandomBytes, sanityCheckOfUniqueness) {
```

Comment:
I would remove this test since we already have other test testing uniqueness of UUIDs I think? (Per other comment I was asking for buffer fill edge case tests).